index.html

```diff
@@ -7,6 +7,7 @@
   <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon">
   <link rel="icon" href="./img/favicon.ico" type="image/x-icon">
   <title>Real Dev Squad</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```

Comment:
@Ankita2002-Fr I do not understand why you use bootstrap just for adding the `status link` in navbar?

Reply:
@ankushdharkar @RohitKuwar 
Actually first i just added simple link  but it did not work for me.It did not show any changes when i run the code.

So that's why i added bootstrap.

But it show problem with deployment,
I will remove it and try to add simply just one link.